src/agora/phy_stats.cc

```diff
@@ -127,6 +134,16 @@ void PhyStats::PrintEvmStats(size_t frame_id) {
   AGORA_LOG_INFO("%s\n", ss.str().c_str());
 }
 
+void PhyStats::RecordEvmSnr(size_t frame_id) {
+  for (size_t i = 0; i < config_->UeAntNum(); i++) {
+    const float evm = evm_buffer_[frame_id % kFrameWnd][i] /
+                      config_->OfdmDataNum();
+    unused(evm);
```

Comment:
Let's wrap these all in an if statement.
if (kEnableCsvLog){}
then you won't have to do the unused.

Reply:
done.